src/app/util/binding-table.cpp

```diff
@@ -23,6 +23,7 @@
 #include <zap-generated/gen_config.h>
 
 static EmberBindingTableEntry bindingTable[EMBER_BINDING_TABLE_SIZE];
+static uint8_t sBindingTableSize;
```

Comment:
Why do we need these changes?  We can keep just having some entries be UNUSED_BINDING and that's fine....